### DISCUSSION TITLE
How to rebuild theme cache?

### DISCUSSION THREAD

**Discussion by mgutz** [NvChad/NvChad#2684] [Q&A] [answered]
I'm manually overriding a few `base_16`  and `base_30` colors per customizing theme article. The overrides don't seem to apply unless I restart nvim, then space+t+h  to reselect the theme. Is there any command to force rebuild of the themes cache?

I'm changing a few properties on the current theme so it exactly matches the terminal colors. Transparency is no good since I don't get nice colors on the status bars, nvimtree.

```lua
M.ui.changed_themes = {
  [M.ui.theme] = {
    base_16 = {
      base00 = "#my_terminal_color",
      ...
    },
  },
}
```

**Comment by siduck** [comment]
just save the file then it'll reload everything.

this will work too

```lua
M.ui.hl_override =  {
  Normal = {  bg = '#000000' }
}
```

**Reply by mgutz** [reply]
Thanks for taking time to answer, and of course NVchad. 

I'm changing the nvimtree sidebar, statusline, and normal bg settings through an external script, not through nvim. When I restart nvim the settings are not applied. I have to go through the steps above.

**Reply by siduck** [reply, marked as answer]
`:lua require('base46').load_all_highlights()` run this in headless

**Reply by mgutz** [reply]
Confirmed it works.  ty
